mailer/template.go

```diff
@@ -268,6 +268,10 @@ func (m TemplateMailer) GetEmailActionLink(user *models.User, actionType, referr
 	switch actionType {
 	case "magiclink":
 		url, err = getSiteURL(referrerURL, globalConfig.API.ExternalURL, m.Config.Mailer.URLPaths.Recovery, "token="+user.RecoveryToken+"&type=magiclink"+redirectParam)
+	case "email_change":
+		url, err = getSiteURL(referrerURL, globalConfig.API.ExternalURL, m.Config.Mailer.URLPaths.Recovery, "token="+user.EmailChangeTokenNew+"&type=email_change"+redirectParam)
+	case "email_change_current":
+		url, err =  getSiteURL(referrerURL, globalConfig.API.ExternalURL, m.Config.Mailer.URLPaths.Recovery, "token="+user.EmailChangeTokenCurrent+"&type=email_change"+redirectParam)
```

Comment:
the filepath argument in `getSiteURL` should be `m.Config.Mailer.URLPaths.EmailChange` instead of `m.Config.Mailer.URLPaths.Recovery`

Reply:
I just missed `getSiteURL` you are right. Second call is happening just when required, yes, one call could be better, I will refactor that.